staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fq/fq_test.go

```diff
@@ -0,0 +1,214 @@
+package fq
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"runtime"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/clock"
+)
+
+// TestPacket is a temporary container for "requests" with additional tracking fields
+// required for the functionality FQScheduler
+type TestPacket struct {
```

Comment:
nit: make it private type, i.e. `testPacket`

Reply:
updated